### PR TITLE
Add document for bucket delayed message tracker metrics

### PR DESCRIPTION
**Pull request by coderzc**
### Documentation

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@mattisonchao Could you please review this PR from a technical perspective? Then writers will review it from a technical writing side. Thank you!

**Review by mattisonchao — APPROVED** [review]

File: versioned_docs/version-3.0.x/reference-metrics.md [hunk 1]
```diff
@@ -201,7 +201,12 @@ All the namespace metrics are labeled with the following labels:
...
-| pulsar_delayed_message_index_size_bytes | Gauge | The total memory size allocated by `InMemoryDelayedDeliveryTracker` of the namespace owned by this broker (in bytes). |
+| pulsar_delayed_message_index_size_bytes | Gauge | The total memory size allocated by `DelayedDeliveryTracker` of the namespace owned by this broker (in bytes). |
+| pulsar_delayed_message_index_bucket_total | Gauge | The number of delayed message index buckets (immutable buckets + LastMutableBucket ) |
+| pulsar_delayed_message_index_loaded | Gauge | The total number of delayed message indexes for in the memory. |
+| pulsar_delayed_message_index_bucket_snapshot_size_bytes | Gauge | The total size of delayed message index bucket snapshot (in bytes). |
+| pulsar_delayed_message_index_bucket_op_count | Counter | The total number of operate delayed message index bucket snapshot. The `state` label can be `succeed`,`failed`,`all` (the `all` means is the total number of all states) and the `type` label can be `create`,`load`,`delete`,`merge`. |
```
Comment (Anonymitaet) [inline versioned_docs/version-3.0.x/reference-metrics.md:208]:
```suggestion
| pulsar_delayed_message_index_bucket_op_count | Counter | The total number of operation delayed message index bucket snapshots. The `state` label can be `succeed`,`failed`, and`all` (`all` means the total number of all states) and the `type` label can be `create`,`load`,`delete`, and `merge`. |
```

File: versioned_docs/version-3.0.x/reference-metrics.md [hunk 2]
```diff
@@ -201,7 +201,12 @@ All the namespace metrics are labeled with the following labels:
...
+| pulsar_delayed_message_index_bucket_op_latency_ms | Histogram | The latency of delayed message index bucket snapshot operation with a given quantile (threshold). The label`type` label can be `create`,`load`,`delete`,`merge`<br/>The label `quantile` can be:<ul><li>quantile="50" is operation latency between (0ms, 50ms]</li><li>quantile="100" is operation latency between (50ms, 100ms]</li><li>quantile="500" is operation between (100ms, 500ms]</li><li>quantile="1000" is operation latency between (500ms, 1s]</li><li>quantile="5000" is operation latency between (1s, 5s]</li><li>quantile="30000" is operation latency between (5s, 30s]</li><li>quantile="60000" is operation latency between (30s, 60s]</li><li>quantile="overflow" is operation latency > 1m</li></ul> |
```
Comment (Anonymitaet) [inline versioned_docs/version-3.0.x/reference-metrics.md:209]:
```suggestion
| pulsar_delayed_message_index_bucket_op_latency_ms | Histogram | The latency of delayed message index bucket snapshot operation with a given quantile (threshold). The `type` label can be `create`,`load`,`delete`, and `merge`<br/>The label `quantile` can be:<ul><li>quantile="50" is operation latency between (0ms, 50ms]</li><li>quantile="100" is operation latency between (50ms, 100ms]</li><li>quantile="500" is operation latency between (100ms, 500ms]</li><li>quantile="1000" is operation latency between (500ms, 1s]</li><li>quantile="5000" is operation latency between (1s, 5s]</li><li>quantile="30000" is operation latency between (5s, 30s]</li><li>quantile="60000" is operation latency between (30s, 60s]</li><li>quantile="overflow" is operation latency > 1m</li></ul> |
```

File: versioned_docs/version-3.0.x/reference-metrics.md [hunk 3]
```diff
@@ -247,10 +252,15 @@ All the topic metrics are labeled with the following labels:
...
 | pulsar_txn_tb_active_total | Gauge | The number of active transactions on this topic. |
 | pulsar_txn_tb_aborted_total | Counter | The number of aborted transactions on the topic. |
 | pulsar_txn_tb_committed_total | Counter | The number of committed transactions on the topic. |
+| pulsar_delayed_message_index_size_bytes | Gauge | The total memory size allocated by `DelayedDeliveryTracker` of the topic owned by this broker (in bytes). |
+| pulsar_delayed_message_index_bucket_total | Gauge | The number of delayed message index buckets (immutable buckets + LastMutableBucket ) |
+| pulsar_delayed_message_index_loaded | Gauge | The total number of delayed message indexes for in the memory. |
+| pulsar_delayed_message_index_bucket_snapshot_size_bytes | Gauge | The total size of delayed message index bucket snapshot (in bytes). |
+| pulsar_delayed_message_index_bucket_op_count | Counter | The total number of operate delayed message index bucket snapshot. The `state` label can be `succeed`,`failed`,`all` (the `all` means is the total number of all states) and the `type` label can be `create`,`load`,`delete`,`merge`. |
```
Comment (Anonymitaet) [inline versioned_docs/version-3.0.x/reference-metrics.md:262]: same as above

File: versioned_docs/version-3.0.x/reference-metrics.md [hunk 4]
```diff
@@ -247,10 +252,15 @@ All the topic metrics are labeled with the following labels:
...
+| pulsar_delayed_message_index_bucket_op_latency_ms | Histogram | The latency of delayed message index bucket snapshot operation with a given quantile (threshold). The label`type` label can be `create`,`load`,`delete`,`merge`<br/>The label `quantile` can be:<ul><li>quantile="50" is operation latency between (0ms, 50ms]</li><li>quantile="100" is operation latency between (50ms, 100ms]</li><li>quantile="500" is operation between (100ms, 500ms]</li><li>quantile="1000" is operation latency between (500ms, 1s]</li><li>quantile="5000" is operation latency between (1s, 5s]</li><li>quantile="30000" is operation latency between (5s, 30s]</li><li>quantile="60000" is operation latency between (30s, 60s]</li><li>quantile="overflow" is operation latency > 1m</li></ul> |
```
Comment (Anonymitaet) [inline versioned_docs/version-3.0.x/reference-metrics.md:263]: same

File: versioned_docs/version-3.0.x/reference-metrics.md [hunk 5]
```diff
@@ -418,6 +428,12 @@ All the subscription metrics are labeled with the following labels:
...
```
Comment (Anonymitaet) [inline versioned_docs/version-3.0.x/reference-metrics.md:436]: same

**Review by Anonymitaet — COMMENTED** [review]
Thanks for your contribution!

1. Have you previewed your changes and ensured everything goes as expected? 

    If not, please [preview your changes locally](https://pulsar.apache.org/contribute/document-preview/#preview-changes) and attach the screenshots to this PR. 

    In this way, you can get your PR merged more quickly. 

2. Does this change apply to the master doc? 

    If so, please update it.

Thank you! 😊

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**